infrastructure/ssz/src/test/java/tech/pegasys/teku/infrastructure/ssz/collections/SszBitvectorTest.java

```diff
@@ -261,4 +261,15 @@ void testFromHexString(final SszBitvector bitvector) {
         SszBitvectorImpl.fromHexString(bitvector.getSchema(), hexString, (int) size);
     SszDataAssert.assertThatSszData(result).isEqualByAllMeansTo(bitvector);
   }
+
+  @ParameterizedTest
+  @MethodSource("bitvectorArgs")
```

Comment:
i'd prefer `bitVectorArgs`

Reply:
well... the class itself follows the `bitvector` naming.. I don't want to pollute this pr with this refactor..